content/docs/deprecated/available-actions.mdx

```diff
@@ -18,7 +18,7 @@ All actions might either be imported directly or via the Actions utility.
 
 ## `init(data: any, schema: JsonSchema, uischema: UISchemaElement, options: InitActionOptions | AJV.Ajv)`
 
-<ApiLink link='core/interfaces/initaction.html' title='API' />
+<ApiLink link='core/interfaces/Actions.InitAction.html' title='API' />
```

Comment:
Do we have to do these adaptions manually every time we move the generated API?

Reply:
I guess so. But I think as this doc is in the deprecated section, we can also just remove it at some point. JF 2.5.x was a very long time ago after all.

Reply:
Alternatively we could just remove the API links in the future but as I already fixed them now, I think we can leave it like this for now.